src/atomizer.js

```diff
@@ -678,11 +684,6 @@ Atomizer.prototype.getCss = function (config/*:AtomizerConfig*/, options/*:CSSOp
         }
     });
 
-    if (options.namespace) {
-        var jssNew = {};
-        jssNew[options.namespace] = jss;
-        jss = jssNew;
-    }
     if (options.helpersNS) {
```

Comment:
@thierryk Helpers can be namespaced if you pass a namespace to it. We talked about this before. The main reason is because of base contextualization, if that happens, then you might want to bump helper classes as well.